build.sh

```diff
@@ -131,11 +131,10 @@ fi
 
 # Build and install the rmm Python package
 if (( NUMARGS == 0 )) || hasArg rmm; then
-    ensureCMakeRan
     cd "${REPODIR}/python"
     if [[ ${INSTALL_TARGET} != "" ]]; then
         echo "building rmm..."
-        python setup.py build_ext --inplace
+        python setup.py build_ext --inplace --library-dir="${LIBRMM_BUILD_DIR}"
```

Comment:
Does this just add the location to the existing library search paths in the setup.py? Concern is if it only searches here it won't find the other needed shared libraries by the Cython like `libcudart` or `libcuda`.

Reply:
This add to the library search path, IE, an additional `-L` to the compiler/linker.